packages/jupyter-ai-magics/jupyter_ai_magics/providers.py

```diff
@@ -46,6 +58,13 @@
 except:
     from pydantic.main import ModelMetaclass
 
+from . import _completion as completion
```

Comment:
Can we rename `_completion.py` to `completion_utils.py`?

Reply:
Done in [5fc1e05](https://github.com/jupyterlab/jupyter-ai/pull/717/commits/5fc1e0582f7c5acc1183fe782e88b238981b1397)